database/migrations/2020_03_27_131152_create_airfield_groups_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateAirfieldGroupsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('airfield_groups', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->string('name');
+        });
+
+        Schema::table('airfields', function (Blueprint $table) {
+            $table->unsignedBigInteger('airfield_group_id')->nullable();
+            $table->foreign('airfield_group_id')->references('id')->on('airfield_groups');
```

Comment:
hmm not a fan of these foreign keys, tend to muck everything up. Any particular reason we want this?